lib/octopus_serverspec_extensions.rb

```diff
@@ -15,3 +16,69 @@
 require 'octopus_serverspec_extensions/matcher/have_windows_line_endings.rb'
 require 'octopus_serverspec_extensions/matcher/have_linux_line_endings.rb'
 require 'octopus_serverspec_extensions/version.rb'
+
+# shared
```

Comment:
how can it be shared... and private at the same time???

Reply:
It's shared between other classes in the module. can't be accessed from outside. Point taken though, redundant comment.